cypress/integration/search-ui.spec.js

```diff
@@ -46,7 +40,7 @@ describe(`${playground} playground test`, () => {
     const select = `.ais-SortBy-select`
     cy.get(select).select('steam-video-games:recommendationCount:asc')
     cy.wait(1000)
-    cy.get(HIT_ITEM_CLASS).eq(0).contains('Rag Doll Kung Fu')
+    cy.get(HIT_ITEM_CLASS).eq(0).contains('Deathmatch Classic')
```

Comment:
Do you understand why this changed between v0.24 and 0.25? I tried to reproduce this behavior to @curquiza but I wasn’t able to :/ 

I saw this happens in the rails integration and here https://github.com/meilisearch/instant-meilisearch/pull/638

Reply:
In this specific case it is because I reduced the dataset significantly. Before that, the tests were made against a remote Meilisearch instance containing 10k games. Now there are only 20 games in total. 

Nonetheless, there have been changes between the two versions: https://github.com/meilisearch/instant-meilisearch/pull/615/files

In my case not sure why, there may be changes sometimes between versions so I didn't think to much of it. But it might be important!

Reply:
Oh, I see.

Let's keep that in mind, maybe the core team could do something to prevent this in the future :D